tools/mutator-utils/mutator.cpp

```diff
@@ -466,12 +482,20 @@ void FunctionMutator::fixAllValues(llvm::SmallVector<llvm::Value *> &vals) {
 llvm::Value *FunctionMutator::getRandomConstant(llvm::Type *ty) {
   if (ty->isIntegerTy()) {
     return llvm::ConstantInt::get(
-        ty->getContext(),
```

Comment:
please try to avoid whitespace changes in pull requests. if we want to reformat the whitespace, that should be done in a totally separate PR

Reply:
I agree. I saw lots of whitespace changes.
You can do `git clang-format` to only format those changed lines after `git add` @isaacdbeans

Reply:
good to know! i ended up clang-formating everything